src/app/client/src/app/modules/dashboard/components/filter/filter.component.spec.ts

```diff
@@ -115,27 +121,41 @@ describe('FilterComponent', () => {
   }));
 
   it('should reset filter', fakeAsync(() => {
+    const spy = spyOn(component, 'resetFilter').and.callThrough();
```

Comment:
Please add relevant assertions for the reset functionality use case.
You're not even calling the reset function method ...

Reply:
resolved